modules/imaging_uploader/jsx/ImagingUploader.js

```diff
@@ -130,28 +141,29 @@ class ImagingUploader extends Component {
         cellStyle.color = '#fff';
         return (
           <td className="label-warning" style={cellStyle}>
-            {cell}
+            {inProgressText}
           </td>
         );
       }
 
       if (cell === 'Success') {
-        const created = row['Number Of Files Created'];
-        const inserted = row['Number Of Files Inserted'];
+        const created = row[filesCreatedKey];
+        const inserted = row[filesInsertedKey];
         return (
           <td style={cellStyle}>
-            {cell} ({inserted} out of {created})
+            {successText} ({inserted} {t('out of',
```

Comment:
inserted and created need to be part of the t() template, not react here. Some languages have them in the other order.